fearless/Common/Storage/Migration/NetworkConnectionsMigrator.swift

```diff
@@ -0,0 +1,20 @@
+import SoraKeystore
+
+final class NetworkConnectionsMigrator: Migrating {
+    private(set) var settings: SettingsManagerProtocol
+
+    init(settings: SettingsManagerProtocol) {
+        self.settings = settings
+    }
+
+    func migrate() throws {
+        let selectedConnection = settings.selectedConnection
+        let deprecatedConnections = ConnectionItem.deprecatedConnections
+        let supportedConnections = ConnectionItem.supportedConnections
+
+        if deprecatedConnections.contains(selectedConnection) {
+            let suitableConnection = supportedConnections.first(where: { $0.type == selectedConnection.type })
```

Comment:
I suggest do nothing if we didn't find suitable connection. We can't use default one because it can be from different network.